apps/courses/src/routes/privacy-policy/index.tsx

```diff
@@ -0,0 +1,178 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Heading,
+  Text,
+  List,
+  ListItem,
+  Divider,
+  Flex,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Icon,
+} from '@chakra-ui/core';
+import Page from '@openmined/shared/util-page';
+import content from '../../content/privacy-policy';
+import GridContainer from '../../components/GridContainer';
+
+import { Link, animateScroll as scroll } from 'react-scroll';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronUp, faCommentAlt } from '@fortawesome/free-solid-svg-icons';
+import CircledNumber from '../../components/CircledNumber';
+
+const SectionListItem = ({ content, index, onClick, ...props }) => (
+  <ListItem cursor="pointer" {...props}>
+    <Link
+      to={`accordion-button-${content.title
+        .replace(/\s+/g, '-')
+        .toLowerCase()}`}
+      onClick={() => onClick(index)}
+      smooth={true}
```

Comment:
What are these next 3 props for? If they're setting the transition CSS information, you should do this the way we do it in all the other components first.

Reply:
It'sa a react-scroll link (we'll get rid of that)